avalon/fusion/pipeline.py

```diff
@@ -135,6 +134,12 @@ def parse_container(tool):
 def get_current_comp():
     """Hack to get current comp in this session"""
     fusion = getattr(sys.modules["__main__"], "fusion", None)
+    if not fusion:
+        try:
+            import BlackmagicFusion as bmf
+            fusion = bmf.scriptapp("Fusion", "localhost")
```

Comment:
We could move this to a more local place as it is a fallback. Yes it is not very reliable but it will be a necessary evil to ensure we can use python scripts through FuScript. For the current state of things I think we can leave it out.